tests/user_examples/extra_src/shared/general_continuum/all_continuum.h

```diff
@@ -34,3 +34,4 @@
 #include "all_continuum_dynamics.h"
 #include "general_continuum.h"
 #include "continuum_particles.h"
+#include "riemann_solver_extra.h"
```

Comment:
This file is deleted in master. It seems that this is not necessary.

Reply:
We do need this. The only difference is that, in the `RiemannSolverExtra`, I used a larger coefficient `η=20d` rather that `3`, to match other people's results related to soil mechanics.

![image](https://github.com/Xiangyu-Hu/SPHinXsys/assets/43446342/2b3f0164-39ca-46c2-9ac1-fb2bb0977146)

If we use the original Riemann solver where `η=3`, the error is way to larger.

Maybe we can make this coefficient `η` as an input parameter.

Reply:
Then, we need make the acoustic riemann solver with a construct parameter with default value 3.0 and different value in your application. We do not need another class.

Reply:
> Then, we need make the acoustic riemann solver with a construct parameter with default value 3.0 and different value in your application. We do not need another class.

This is done. Please check again.

Reply:
@Shuaihao-Zhang @Xiangyu-Hu 
The parameter may not be passed in fuction "DissipativePJump(const Real &u_jump, const Real coeff = 3.0)". 

The limiter coeff should be a member of Acoustic Riemann solver with default value limiter_coeff_ = 3.0 and reset it when the a RiemannSolver object is instanced, for example in your case

template <class RiemannSolverType>
BaseStressRelaxation1stHalf<RiemannSolverType>::BaseStressRelaxation1stHalf(BaseInnerRelation& inner_relation)
		: BaseRelaxationPlastic(inner_relation), riemann_solver_(plastic_continuum_, plastic_continuum_ , limiter_coeff),
			velocity_gradient_(particles_->velocity_gradient_) 
{}

In this case, you may need introduce second constructor for RiemannSolver class. 
This may be one more proper solution.

Reply:
> @Shuaihao-Zhang @Xiangyu-Hu The parameter may not be passed in fuction "DissipativePJump(const Real &u_jump, const Real coeff = 3.0)".
> 
> The limiter coeff should be a member of Acoustic Riemann solver with default value limiter_coeff_ = 3.0 and reset it when the a RiemannSolver object is instanced, for example in your case
> 
> template BaseStressRelaxation1stHalf::BaseStressRelaxation1stHalf(BaseInnerRelation& inner_relation) : BaseRelaxationPlastic(inner_relation), riemann_solver_(plastic_continuum_, plastic_continuum_ , limiter_coeff), velocity_gradient_(particles_->velocity_gradient_) {}
> 
> In this case, you may need introduce second constructor for RiemannSolver class. This may be one more proper solution.

@ChiZhangatTUM  Thanks. This is done. Please check.